components/automate-ui/src/app/modules/roles/list/roles-list.component.html

```diff
@@ -38,9 +38,11 @@
                   <chef-table-cell></chef-table-cell>
                   <chef-table-cell>{{ role.type | iamType }}</chef-table-cell>
                   <chef-table-cell class="three-dot-column">
+                    <!-- <app-authorized [allOf]="[`/iam/v2beta/roles/{id}`, 'delete', role.id]"> -->
```

Comment:
Still part of the consistency cleanup, by adding this even as commented out code, it helps us remember that the same code around the control menu should go here eventually.

Reply:
Heh I was gonna ask for the opposite. We don't even know if this will look like and we know it needs to happen everywhere. Commented code sitting around is a pet peeve of mine though so w/e works.